cmd/commands/docs.go

```diff
@@ -34,19 +34,23 @@ func Docs(rootCmd *cobra.Command) *cobra.Command {
 		Short:  "generate riff command documentation",
 		Hidden: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-
-			fi, err := os.Stat(directory)
-			if os.IsNotExist(err) {
-				if err := os.Mkdir(directory, 0744); err != nil {
-					return err
-				}
-			} else if !fi.Mode().IsDir() {
-				return fmt.Errorf("path %q already exists but is not a directory", directory)
-			}
-			return doc.GenMarkdownTree(rootCmd, directory)
-
+			return GenerateDocs(rootCmd, directory, fs)
 		},
 	}
 	docsCmd.Flags().StringVarP(&directory, "dir", "d", "docs", "the output directory for the docs.")
 	return docsCmd
 }
+
+func GenerateDocs(rootCommand *cobra.Command, directory string, fs Filesystem) error {
+	fileinfo, err := fs.Stat(directory)
+	if os.IsNotExist(err) {
+		if err = fs.Mkdir(directory, 0744); err != nil {
```

Comment:
Might as well call this (and use in the underlying implementation) `MkDirAll`

Reply:
The UI does not show anything but it's now done (cf. status.github.com, they currently have DB issues)